src/command_modules/azure-cli-acr/README.rst

```diff
@@ -0,0 +1,183 @@
+        --new-sp                      : Create a new service principal. If provided, no --app-id should
+                                        be specified. Optional: Use -p to specify a password.
+        --password -p                 : Password used to log into a container registry.
+        --role -r                     : Name of role. Allowed roles: owner, contributor, reader.
+                                        Default: reader.
+        --storage-account-name -s     : Name of new or existing storage account. If not provided, a
+                                        random storage account name will be generated.
+
+    Examples
+        Create a container registry with a new storage account
+            az acr create -n myRegistry -g myResourceGroup -l southus
+        Create a container registry with an existing storage account
+            az acr create -n myRegistry -g myResourceGroup -l southus -s myStorageAccount
+        Create a container registry with a new service principal
+            az acr create -n myRegistry -g myResourceGroup -l southus --new-sp -p myPassword -r Owner
+        Create a container registry with an existing service principal
+            az acr create -n myRegistry -g myResourceGroup -l southus --app-id myAppId -r Owner
+
+Delete a container registry
+-------------
+::
+
+    Command
+        az acr delete: Delete a container registry.
+
+    Arguments
+        --name -n [Required]: Name of container registry.
+        --resource-group -g : Name of resource group.
+
+List container registries
+-------------
+::
+
+    Command
+        az acr list: List container registries.
+
+    Arguments
+        --resource-group -g: Name of resource group.
+
+    Examples
+        List container registries and show result in a table
+            az acr list -o table
+        List container registries in a resource group and show result in a table
+            az acr list -g myResourceGroup -o table
+
+Get a container registry
+-------------
+::
+
+    Command
+        az acr show: Get a container registry.
+
+    Arguments
+        --name -n [Required]: Name of container registry.
+        --resource-group -g : Name of resource group.
+
+Update a container registry
+-------------
+::
+
+    Command
+        az acr update: Update a container registry.
+
+    Arguments
+        --name -n [Required]: Name of container registry.
+        --app-id            : The app id of an existing service principal. If provided, no --new-sp or
+                            -p should be specified.
+        --disable-admin     : Disable admin user.
+        --enable-admin      : Enable admin user.
+        --new-sp            : Create a new service principal. If provided, no --app-id should be
+                            specified. Optional: Use -p to specify a password.
+        --password -p       : Password used to log into a container registry.
+        --resource-group -g : Name of resource group.
```

Comment:
Our registry names are unique and we don't need resource group to query ARM for a registry as long as the registry in under the user's current subscription. If the user provides resource group, we will use it, otherwise, we will query ARM for the full resource id. I would tend to remove it, but for consistency reason we made it an optional argument.